localstripe/resources.py

```diff
@@ -2064,13 +2064,17 @@ class Subscription(StripeObject):
     _id_prefix = 'sub_'
 
     def __init__(self, customer=None, metadata=None, items=None,
+                 plan=None, quantity=None, # optional "items" dictionary
```

Comment:
I don't see these parameters in Stripe official docs: https://stripe.com/docs/api/subscriptions/update

Isn't this an old deprecated way of updating a subscription?

Reply:
The functionality is supported by the 1st party stripe-php sdk: https://github.com/stripe/stripe-php/blob/0e24d89888e4bf051003c45c447b32e5298df55c/lib/Subscription.php#L36

Simply update the quantity directly on the model and `save()`. 

This is also used by the 1st party Laravel package 'Cashier' which our project uses:

https://github.com/laravel/cashier/blob/39916c390454ff21cf25e28fb517eac6f13bd4bd/src/Subscription.php#L370-L376 

For these reasons I'd say there is no sign of it being deprecated until their own SDK stops supporting it :-)